### PR TITLE
fix udev build error

### LINKED ISSUE #52: You need to update udev from shared-modules
In order to fix [udev build](https://flathub.org/builds/#/builders/32/builds/968) you need to update [shared-modules](https://github.com/flathub/shared-modules) submodule (specifically [this commit](https://github.com/flathub/shared-modules/commit/65f2d48216b74c4722a1bc68f6acc3dac70bb910))

### PR DESCRIPTION
**Pull request by Rechi**
fixes #52

### CODE REVIEW

**Comment by tilosp** [comment]
bot, build

**Comment by wsnipex** [comment]
oh great, now jpeg is missing for pillow. I wonder how this worked before

File: deps/zlib.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "zlib",
```
Comment (Erick555) [inline deps/zlib.json]: zlib is available in 18.08 runtime

File: deps/libjpeg-turbo.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "libjpeg-turbo",
```
Comment (Erick555) [inline deps/libjpeg-turbo.json]: This is also available on runtime.

**Comment by Erick555** [comment]
You may look at how others build [python-pillow](https://github.com/flathub/org.gnome.Lollypop/blob/3575a7d7f2ca6c18f86fbe78ffa2e776452ddc68/org.gnome.Lollypop.json#L91). Perhaps you need openjpeg dep instead of libjpeg-turbo.

**Comment by Erick555** [comment]
The build worked, there was just random flathub error for i386.

**Comment by Rechi** [comment]
@wsnipex can you merge this PR?

**Comment by wsnipex** [comment]
thanks!

**Pull request merged**